src/app/global/globals.css

```diff
@@ -26,6 +26,27 @@ body {
     rgb(var(--background-start-rgb));
 }
 
+@layer base {
+  h1 {
+    @apply text-2xl font-bold sm:text-2xl md:text-3xl lg:text-4xl;
+  }
+  h2 {
+    @apply text-2xl font-bold sm:text-xl md:text-2xl lg:text-3xl;
+  }
+  h3 {
+    @apply text-xl font-bold sm:text-lg md:text-xl lg:text-2xl;
+  }
+  h4 {
+    @apply text-lg font-bold sm:text-base md:text-lg lg:text-xl;
+  }
+  h5 {
+    @apply text-base font-bold sm:text-sm md:text-base lg:text-lg;
+  }
+  h6 {
+    @apply text-sm font-bold sm:text-xs md:text-sm lg:text-base;
```

Comment:
Pourquoi pas d'italique ?

Reply:
![image](https://github.com/The-Geneps-Personnal-Project/EsportsStats-FrontWeb/assets/71601150/522c83be-04cb-43c6-834e-b766f6f2db82)
Pour ça